src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -960,6 +959,13 @@ public async Task ReadAsStreamAsync_HandlerProducesWellBehavedResponseStream(boo
             await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
             {
                 var request = new HttpRequestMessage(HttpMethod.Get, uri) { Version = UseVersion };
+                if (PlatformDetection.IsBrowser)
+                {
+#if !NETFRAMEWORK
+                    request.Options.Set(new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse"), true);
```

Comment:
So now we're only testing with WebAssemblyEnableStreamingResponse = true?  I think we want to test both cases.

so when WebAssemblyEnableStreamingResponse == true sync reads won't work and and the stream is not seekable
WebAssemblyEnableStreamingResponse == false sync reads do work and the stream is seekable